packages/web-frontend/src/containers/mobile/MapOverlayBar/MapOverlayLibrary.js

```diff
@@ -13,7 +13,7 @@ import Button from '@material-ui/core/Button';
 
 import { setMapOverlays, clearMeasure } from '../../../actions';
 import { selectCurrentMapOverlayCodes } from '../../../selectors';
-import { MAP_OVERLAY_SELECTOR, LEAFLET_Z_INDEX } from '../../../styles';
+import { MAP_OVERLAY_SELECTOR, MOBILE_BACKGROUND_COLOR, LEAFLET_Z_INDEX } from '../../../styles';
```

Comment:
There are a few unrelated changes in this PR just tweaking the background colour of most mobile components to match the designs